cmd/copyurl/copyurl.go

```diff
@@ -61,10 +65,14 @@ output to be written to standard output.
 			}
 		}
 		cmd.Run(true, true, command, func() error {
+			var dst fs.Object
 			if stdout {
 				err = operations.CopyURLToWriter(context.Background(), args[0], os.Stdout)
 			} else {
-				_, err = operations.CopyURL(context.Background(), fsdst, dstFileName, args[0], autoFilename, noClobber)
+				dst, err = operations.CopyURL(context.Background(), fsdst, dstFileName, args[0], autoFilename, noClobber)
+				if printFilename && err == nil && dst != nil {
+					fmt.Printf("%s", dst.Remote())
```

Comment:
I think you want to print a newline here too - that is the unix way. So something like

    fmt.Println(dst.Remote())

Reply:
Sure. I had it, amended it away, and now amended it back in! :)